src/app/components/LanguageNameInput.svelte

```diff
@@ -0,0 +1,81 @@
+<script lang="ts">
+  import { onMount, createEventDispatcher } from "svelte";
+  import AutoComplete from "simple-svelte-autocomplete";
+
+  import worker from "../spawn-worker";
+  import type { KeyboardMetadata } from "@common/types";
+
+  export let label = "";
+  export let subtext = "";
+  export let bold: boolean = true;
+  export let cyData = "autocomplete-label";
+
+  // To store selected language
+  // TODO: rename to selectedLanguage
+  export let selected: KeyboardMetadata | undefined = undefined;
```

Comment:
Could this be something that can be taken care of from this PR?

Reply:
Yes! Honestly, I just forgot >.<